src/Tools/dotnet-user-jwts/src/Helpers/ConsoleTable.cs

```diff
@@ -59,25 +57,94 @@ public void Write()
                 .Select(x => x!.ToString()!.Length).Max())
             .ToList();
 
+        var equalColumnLengths = (Console.WindowWidth / _columns.Count);
```

Comment:
We should have a minimum here as well e.g. `Math.Max(Console.WindowWidth / _columns.Count, 5);`